api/proto/core/pipeline/cron/cron.proto

```diff
@@ -47,6 +47,41 @@ service CronService {
       put: "/api/pipeline-crons/{cronID}",
     };
   }
+
+  // edge
+  rpc CronInsertOrUpdate (CronInertOrUpdateRequest) returns (CronInertOrUpdateResponse) {
+    option (google.api.http) = {
+      post: "/api/pipeline-crons/actions/insert-or-update",
```

Comment:
Just `CronCreate` is enough on API level.
Otherwise, invoker will always invoke `insert-or-update`.